core/templates/dev/head/pages/exploration_player/ImagePreloaderService.js

```diff
@@ -100,7 +102,7 @@ oppia.factory('ImagePreloaderService', [
         if (_imageLoadedCallback[loadedImage.filename]) {
           var onLoadImageResolve = (
             (_imageLoadedCallback[loadedImage.filename]).resolveMethod);
-          _getUrlUsingFileInCache(loadedImage.filename, onLoadImageResolve);
+          _getImageUrl(loadedImage.filename, onLoadImageResolve);
```

Comment:
Can we do
```javascript
var objectUrl = URL.createObjectURL(loadedImageFile.data);
onLoadCallback(objectUrl);
```
here directly? Since, we already have the file data and we don't need to call `AssetsBackendApiService` again. I know that it's loaded from cache, but even that is unnecessary.

Reply:
Done!